.hadolint.yml

```diff
@@ -0,0 +1,5 @@
+ignored:
+ - DL3009
```

Comment:
Can you add comments explaining what these ignored errors are?

Reply:
Handled all of the errors so I removed this file.